.gitattributes

```diff
@@ -0,0 +1,5 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
```

Comment:
we found that this gave us the best spotless experience for developers on windows: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/707/commits/caa335d1108ddc126db87d7558617f7148f6b67e

Reply:
updated